test/extensions/INTEL/SPV_INTEL_task_sequence/task_sequence.ll

```diff
@@ -80,9 +80,9 @@ define weak_odr dso_local spir_kernel void @_ZTS8MyKernel(ptr addrspace(1) nound
 entry:
   %myMultTask.i = alloca %"class.sycl::_V1::ext::intel::experimental::task_sequence.3", align 8
   store i32 0, ptr %myMultTask.i, align 8, !tbaa !10
-; CHECK-LLVM: %[[TSCreate:[a-z0-9.]+]] = call spir_func target("spirv.TaskSequenceINTEL") @_Z66__spirv_TaskSequenceCreateINTEL_RPU3AS125__spirv_TaskSequenceINTELPiiiii(ptr @_Z4multii, i32 10, i32 -1, i32 17, i32 1)
+; CHECK-LLVM: %[[TSCreate:[a-z0-9.]+]] = call spir_func target("spirv.TaskSequenceINTEL") @_Z66__spirv_TaskSequenceCreateINTEL_RPU3AS125__spirv_TaskSequenceINTELPiiiii(ptr @_Z4multii, i32 10, i32 -1, i32 17, i32 0)
```

Comment:
Should the the GetCapacity field also be tested to see if it can accept 0?

Reply:
Applied